### PR TITLE
Adding cmdlet to prune backup external store

### PR DESCRIPTION
**Pull request by poetw**
Adding cmdlet to prune backup external store

### CODE REVIEW

File: src/Azs.Backup.Admin/Azs.Backup.Admin.psd1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@{
+  GUID = '6b8410e2-074e-44c1-81f1-410accf696a5'
+  RootModule = 'Azs.Backup.Admin.psm1'
+  ModuleVersion = '0.9.0'
```
Comment (bganapa) [inline src/Azs.Backup.Admin/Azs.Backup.Admin.psd1:4]: module version needs update

**Review by bganapa — APPROVED** [review]

File: src/Azs.Backup.Admin/Azs.Backup.Admin.psd1 (hunk 1 above)
Comment (poetw) [reply src/Azs.Backup.Admin/Azs.Backup.Admin.psd1:4]: Hi @bganapa, this is one of the files that were generated. Should I exclude all of them from the PR?

Comment (bganapa) [reply src/Azs.Backup.Admin/Azs.Backup.Admin.psd1:4]: Ah didnt realize that, Can you pls make a PR to remove all the generated files?

Comment (poetw) [reply src/Azs.Backup.Admin/Azs.Backup.Admin.psd1]: Hi @bganapa, thanks for confirming. Updated the PR to remove generated files.

File: src/Azs.Backup.Admin/examples/Invoke-AzsPruneBackupLocationExternalStore.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
```
Comment (bganapa) [inline src/Azs.Backup.Admin/examples/Invoke-AzsPruneBackupLocationExternalStore.md:2]: Can you pls add examples here?

File: src/Azs.Backup.Admin/examples/Invoke-AzsPruneBackupLocationExternalStore.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
+```
+
+{{ Add description here }}
+
+### Example 2: {{ Add title here }}
+```powershell
```
Comment (bganapa) [inline src/Azs.Backup.Admin/examples/Invoke-AzsPruneBackupLocationExternalStore.md:11]: @poetw once you add examples here, if you build again, it should update the docs folder contents to have the examples added

**Review by bganapa — CHANGES_REQUESTED** [review]
docs update

**Review by bganapa — APPROVED** [review]

**Comment by poetw** [comment]
Hi @TheOnlyWei, could you help review this PR? Thanks!

**Review by bganapa — APPROVED** [review]

**Review by TheOnlyWei — APPROVED** [review]

**Pull request merged**